src/lib/storage.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+const { DynamoDBClient, GetItemCommand, PutItemCommand, BatchWriteItemCommand } = require('@aws-sdk/client-dynamodb')
+const { NodeHttpHandler } = require('@aws-sdk/node-http-handler')
+const { marshall, unmarshall } = require('@aws-sdk/util-dynamodb')
```

Comment:
Is this right, are these typos in the dependency?

```suggestion
const { marshal, unmarshal } = require('@aws-sdk/util-dynamodb')
```

Reply:
I'm the very last person that should talk about English grammar, I think `marshall` is American and `marshal` is British - same as `color` and `colour`